src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
> Sorry; I feel the question is constantly changing

I always tried to directly refer to your previous comments and not introduce new topics into the discussion. Can you please give examples of posts in this thread where I did not refer to your comments to support your claim and if so to help me avoid that in the future? The other way around I find that you mentioned topics like escaping captures/SEE (ok, at least partly related), ttcaptures, testing conditions/bounds, other engines/search techniques, atomic chess, "single mistake" issues, upstream changes, etc. So maybe I ask new questions because you introduce new topics?

> While I have concerns about "how in general for any patch to identify whether it is simple"

May I ask where you are quoting from? I did not ask you to judge whether a patch is simple, just whether it is good or bad (not only performance-wise to avoid misunderstandings).